openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $# -eq 0 ]; then
+    echo "$0: usage: rerun_0006.sh <arguments>. At minimum, '--settings=<environment>' is expected."
+    exit 1
+fi
+
+./manage.py lms migrate course_groups 0005 --fake "$@"
+./manage.py lms migrate course_groups 0006 "$@"
```

Comment:
Never mind, you're correct.

On Fri, Oct 30, 2015 at 11:15 AM, Eric Fischer <notifications@github.com>
wrote:

> In openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh
> <https://github.com/edx/edx-platform/pull/10295#discussion_r43513157>:
>
> > @@ -0,0 +1,8 @@
> > +#!/bin/bash
> > +if [ $# -eq 0 ]; then
> > +    echo "$0: usage: rerun_0006.sh <arguments>. At minimum, '--settings=<environment>' is expected."
> > +    exit 1
> > +fi
> > +
> > +./manage.py lms migrate course_groups 0005 --fake "$@"
> > +./manage.py lms migrate course_groups 0006 "$@"
>
> Why is that? (I'm not super well-versed in bash scripting in general).
>
> Each positional argument should be split into its own separate word:
> http://www.gnu.org/software/bash/manual/bash.html#Special-Parameters
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10295/files#r43513157>.
>